ansible/roles/hand_prod/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Checkout or create the config branch
+  shell: git checkout -B '{{ config_branch }}'
+         chdir={{ros_workspace}}/src/sr_config
+  sudo: yes
+  sudo_user: "{{ros_user}}"
```

Comment:
simply getting / creating the branch. We're not pushing it or anything, that'll be done by @jlaforga once the tuning has been done.